src/fix/fixCodec.cpp

```diff
@@ -0,0 +1,737 @@
+#include "fixCodec.h"
+
+namespace neueda
+{
+
+bool
+fixCodec::getFixFieldFromNode (xmlNode* field, fixField& f)
+{
+    string name;
+    string req;
+    string type;
+    bool required;
+    int64_t tag;
+
+    name.assign ((const char*)xmlGetProp (field, (const xmlChar*)"name"));
+    req.assign ((const char*)xmlGetProp (field, (const xmlChar*)"required"));
+
```

Comment:
You can move those to the string ctor, rather than using assign since you know field and name at start of method